deployment/ccip/changeset/deploy.go

```diff
@@ -422,6 +380,113 @@ func DeployCCIPContracts(
 	return nil
 }
 
+type USDCConfig struct {
+	EnabledChains []uint64
+	USDCAttestationConfig
+	CCTPTokenConfig map[cciptypes.ChainSelector]pluginconfig.USDCCCTPTokenConfig
+}
+
+func (cfg USDCConfig) EnabledChainMap() map[uint64]bool {
+	m := make(map[uint64]bool)
+	for _, chain := range cfg.EnabledChains {
+		m[chain] = true
+	}
+	return m
+}
+
+type USDCAttestationConfig struct {
+	API         string
+	APITimeout  *commonconfig.Duration
+	APIInterval *commonconfig.Duration
+}
+
+type DeployCCIPContractConfig struct {
```

Comment:
since this is top level changeset input config, lets put it in the changeset file itself